.gitignore

```diff
@@ -0,0 +1 @@
+demo-pipeline/terraform/.terraform.lock.hcl
```

Comment:
Can't this be covered by the `demo-pipeline/.gitignore` file?

Reply:
It can, but Hashicorp recommends `.terraform.lock.hcl` be committed.  That being said, since it is a demonstration, I don't think it is necessary